Meridian59.DebugUI/CustomDataGridColumns/TagColumn.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+namespace Meridian59.DebugUI.CustomDataGridColumns
+{
+    class TagCell : DataGridViewTextBoxCell
```

Comment:
Does this class have any purpose yet or is it a preparation for something you have in mind ?